docs/monitoring/historyserver.zh.md

```diff
@@ -22,62 +22,62 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-Flink has a history server that can be used to query the statistics of completed jobs after the corresponding Flink cluster has been shut down.
+Flink 自带的 history server 可以在已执行完作业对应的 Flink 集群关闭之后查询该作业的统计信息。
 
-Furthermore, it exposes a REST API that accepts HTTP requests and responds with JSON data.
+除此之外，它还提供了一个 REST API，可以通过 HTTP 以 JSON 格式发送和接收数据。
 
 * This will be replaced by the TOC
 {:toc}
 
-## Overview
+## 概览
 
-The HistoryServer allows you to query the status and statistics of completed jobs that have been archived by a JobManager.
+History server 可以查询被 JobManager 存档的已执行完作业的状态和统计信息。
 
-After you have configured the HistoryServer *and* JobManager, you start and stop the HistoryServer via its corresponding startup script:
+在配置完 History server 和 JobManager 之后，用户可以通过下面的启动脚本来开启和关停 History server：
 
 {% highlight shell %}
-# Start or stop the HistoryServer
+# 开启或关停 History Server
 bin/historyserver.sh (start|start-foreground|stop)
 {% endhighlight %}
 
-By default, this server binds to `localhost` and listens at port `8082`.
+默认情况下，History server 绑定到本机 `localhost` 的 `8082` 端口。
 
-Currently, you can only run it as a standalone process.
+目前你可以把它当做单独的进程来运行。
 
-## Configuration
+## 配置
 
-The configuration keys `jobmanager.archive.fs.dir` and `historyserver.archive.fs.refresh-interval` need to be adjusted for archiving and displaying archived jobs.
+存档和展示已完成的作业 需要调配 `jobmanager.archive.fs.dir` 和 `historyserver.archive.fs.refresh-interval` 这俩配置项。
 
 **JobManager**
 
-The archiving of completed jobs happens on the JobManager, which uploads the archived job information to a file system directory. You can configure the directory to archive completed jobs in `flink-conf.yaml` by setting a directory via `jobmanager.archive.fs.dir`.
+JobManager 会进行已完成作业的存档，把已存档作业的信息上传到一个文件系统目录上。用户可以通过设置 `jobmanager.archive.fs.dir` 来配置这个存档目录，将 `flink-conf.yaml` 中已完成的作业都存档下来。
 
 {% highlight yaml %}
-# Directory to upload completed job information
+# 已完成作业信息的上传目录
 jobmanager.archive.fs.dir: hdfs:///completed-jobs
 {% endhighlight %}
 
 **HistoryServer**
 
-The HistoryServer can be configured to monitor a comma-separated list of directories in via `historyserver.archive.fs.dir`. The configured directories are regularly polled for new archives; the polling interval can be configured via `historyserver.archive.fs.refresh-interval`.
+History Server 可以监控 `historyserver.archive.fs.dir` 配置的用逗号分隔的文件目录列表。并且会为新存档定期轮询已配置的目录，轮询的间隔可以通过 `historyserver.archive.fs.refresh-interval` 来配置。
```

Comment:
```suggestion
History Server 可以监控 `historyserver.archive.fs.dir` 配置的用逗号分隔的文件目录列表。History Server 会定期轮询配置的目录以发现新存档，轮询的间隔可以通过 `historyserver.archive.fs.refresh-interval` 来配置。
```

Reply:
@wuchong Thanks for your comments.  It's more reasonable and fluent. I have optimized the translation.